test/l4lb/test.sh

```diff
@@ -145,4 +145,6 @@ for i in $(seq 1 10); do
 done
 
 # Cleanup
-kind delete cluster
+docker rm -f lb-node
+docker rm -f nginx
+docker network rm cilium-l4lb
```

Comment:
May this test break any consequent test if we don't clean the docker setup? If yes, then we better do a `trap EXIT` to cleanup